app/components/offering-editor.js

```diff
@@ -0,0 +1,199 @@
+/* global moment */
```

Comment:
moment should now be used as 
`import moment from 'moment';`
and not as a global